src/mapgen.cpp

```diff
@@ -2292,7 +2314,7 @@ bool mapgen_function_json_base::setup_common( JsonObject jo )
     objects.load_objects<jmapgen_nested>( jo, "place_nested" );
     objects.load_objects<jmapgen_graffiti>( jo, "place_graffiti" );
     objects.load_objects<jmapgen_translate>( jo, "translate_ter" );
-
+    objects.load_objects<jmapgen_faction>( jo, "faction_owner" );
```

Comment:
Maybe add a comment that this has to be the last object to load, because it has to modify other placed items.